TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>We should talk about how you picked this particular example. In a sense, it does demonstrate that you shouldn't just trust a visual inspection without cleaning out the "bad" data.</p><p> </p><p>However, I think we should at least look for a more misleading false positive. This will be tricky, but you could look at the <a href="https://tev.mit.edu/data/collection/193/" target="_blank">TESS TOI</a> (target of interest) list, which has flagged false positives. In your position, I would create a "pipeline" which takes the above code, goes through some of the false positives on the TOI list, and generates many phase-folded graphs (via the intermediate step of creating a periodogram). </p><p> </p><p>Ideally, we would find an example that is "cleared up" by checking against the quality flags. If necessary, we can just find an example that required follow-up observations to definitively disprove the presence of a planet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>okay, i found a false positive example from that catalog and was able to use the data quality flags to prove the dip is not a transit. see next push. </p>